src-tauri/src/main.rs

```diff
@@ -34,28 +50,69 @@ fn main() {
         assert!(status.success());
     }
 
-    let (mut rx, child) = Command::new_sidecar("ad4m")
-        .expect("Failed to create ad4m command")
-        .args(["serve"])
-        .spawn()
-        .expect("Failed to spawn ad4m serve");
-
-    tauri::async_runtime::spawn(async move {
-        while let Some(event) = rx.recv().await {
-            match event.clone() {
-                CommandEvent::Stdout(line) => log::info!("{}", line),
-                CommandEvent::Stderr(line) => log::info!("{}", line),
-                _ => log::info!("{:?}", event),
-            }
-        }
-    });
+    let free_port_clone = free_port.clone();
+
+    let url = app_url(free_port);
```

Comment:
If the `app_url` only used for logging, I don't the value to keep it..

Reply:
This would be used if we were using a splash screen because, because for the splash screen we using an HTML file but for the main window we need to use the local dev URL for the app so we can't rely on tauri to provide it separately fro prod & dev.